src/Services/userService.ts

```diff
@@ -88,13 +64,19 @@ export const getParentByEmailAndPassword = async (
   res: Response
 ) => {
   try {
-    const response = await userRepo.findOneByEmailAndPassword(
-      userLogin.email,
-      userLogin.password
-    );
+    const response = await userRepo.findOneByEmail(userLogin.email);
 
     if (!response) {
-      return failed('user');
+      return failed(new Error('Email or password is incorrect'));
+    }
+
+    const isPsswordCorrect = await comparePassword(
```

Comment:
🤨

Reply:
??

Reply:
🚨 URGENT: TYPO ALERT 🚨

Hey fellow devs, gather 'round for a tale of epic proportions! 📜

So, picture this: A valiant coder, fueled by caffeine and determination, submits a pull request, ready to conquer the coding battleground. But lo and behold! A sneaky typo manages to slip past the vigilant eyes of our brave reviewer.

🕵️‍♂️ Detective dev squad, assemble! 🕵️‍♀️

A typo, so cunning and sly, it tiptoed through the pull request like a ninja in the night. 😱 Now, we face the aftermath, standing on the precipice of chaos.

🆘 SOS, fellow code warriors! 🆘

Let this be a reminder to always stay vigilant, for typos lurk in the shadows, waiting to strike when you least expect it. ⚔️ Brace yourselves, for the battle against the elusive typo has begun!

👾 May your code be typo-free and your pull requests pristine, my fellow developers! 👾 #CodeWarriors #TypoGate #DeveloperChronicles

Reply:
🫠